common/uint160.go

```diff
@@ -94,6 +94,14 @@ func (f Uint160) MarshalJSON() ([]byte, error) {
 	return []byte("\"" + str + "\""), err
 }
 
+func (f Uint160) UnmarshalJSON(in []byte) (err error) {
+	if len(in) < 2 {
+		return errors.New("[Common]: Uint160UnmarshalJSON err, len < 2")
+	}
+	f, err = ToScriptHash(string(in[1 : len(in)-1]))
```

Comment:
This will not update f. You need to change
```func (f Uint160)```
to
```func (f *Uint160)```
then change `f, err = ` to `temp, err :=`,
and finally call `f.SetBytes(temp.ToArray())` to update f.